### PR TITLE
Include scss style in webpack output

### PR DESCRIPTION
**Pull request by artonge**

### CODE REVIEW

File: src/styles/styles.scss [hunk 1]
```diff
@@ -61,13 +61,6 @@
 		flex-direction: column;
 	}
 
-	.emptycontent {
-		padding: 50px 0;
-		margin: 0;
-
-		@include icon-black-white('notifications', 'notifications', 1);
-	}
-
```
Comment (artonge) [inline src/styles/styles.scss:70]: Useless as we use vue component for that

File: src/styles/styles.scss [hunk 2]
```diff
@@ -196,18 +189,18 @@
 				&:active {
 					box-shadow: 0 0 0 2px var(--color-main-text);
 				}
+			}
 
-				&--tabbed {
-					box-shadow: 0 0 0 2px var(--color-main-text);
-					background-color: var(--color-primary-light-hover);
+			&--tabbed:not(.primary) {
```
Comment (artonge) [inline src/styles/styles.scss:194]: Was causing issue with compilation, simply rearranged

**Comment by nickvergessen** [comment]
You can ignore the performance warning for now

File: src/styles/styles.scss (hunk 2 above)
Comment (nickvergessen) [reply src/styles/styles.scss:194]: Now it's also affecting a-href links and seems to not look good anymore (it becomes square)?

Also the colors seem to fall apart
Bildschirmfoto vom 2022-06-02 12-33-45

Comment (artonge) [reply src/styles/styles.scss:194]: Issue already exist outside of this PR.
Fix here: https://github.com/nextcloud/server/pull/32713

**Review by Pytal — APPROVED** [review]

**Pull request merged**